rosdep/base.yaml

```diff
@@ -4415,7 +4415,7 @@ qtbase5-dev:
   debian: [qtbase5-dev]
   fedora: [qt5-qtbase-devel]
   freebsd: [qt5-core]
-  gentoo: ['dev-qt/qtcore:5']
+  gentoo: ['dev-qt/qtcore:5', 'dev-qt/widgets:5', 'dev-qt/qttest:5']
```

Comment:
This should be `dev-qt/qtwidgets:5`, and I don't think `dev-qt/qttest:5` is needed unless they have the `test` `USE` flag enabled...

@awesomebytes Could you please fix this typo?

Reply:
@allenh1 good catch on the typo. But about qttest, it is actually needed even with the disabled test USE flag by higher level packages (as I arised in this issue: https://github.com/ros/ros-overlay/issues/592). It is just expected to be there in qt by further ROS packages as the other OS install it all together.

Reply:
PR about the typo: https://github.com/ros/rosdistro/pull/19791
Sorry about it.